docs/contribution_guide/ways_to_contribute.rst

```diff
@@ -0,0 +1,17 @@
+Ways to contribute
+==================
+
+There are many different ways to contribute to the Fairlearn community, for
+example:
+
+* contributing code to add/enhance
+    * unfairness mitigation techniques
+    * fairness metrics
+    * visualizations 
+* writing example notebooks
+* improving documentation
+    * in the code through docstrings
+    * for the website through user guides, etc.
+* reporting bugs
+* requesting new features
+* writing proposals for larger changes
```

Comment:
I'm thinking that it might look visually better if the bullets are written as sentences in imperative:
```
* Contribute code in the following areas:

  * unfairness mitigation algorithms
  * fairness metrics
  * visualizations

* Write example notebooks showcasing

  * case studies in new domains
  * usage of Fairlearn components

* Improve documentation, including:

    * docstrings
    * website content

* Report bugs.
* Request new features.
* Write proposals for larger changes.
```